sms.js

```diff
@@ -0,0 +1,11 @@
+const accountSid = "AC6e4a0c264045033cb9dd3df126bedbcc";
+const authToken = "42085f49dc665abd66771ea32795793d"; 
```

Comment:
Don't forget to remove this keys, and update the keys in your Twilio account.

```
const accountSid = "X";
const authToken = "X"; 
```

Reply:
Auth token has been rotated.